starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,70 @@
+body {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 20px;
+}
+
+h1 {
+      font-size: 30px;
+
+}
+
+.apple-pie-picture {
+    background-image: url(../images/apple-pie.jpg);
```

Comment:
This is a relative path.  This one works.  This is the way you should link things in your HTML & CSS files.